README.md

```diff
@@ -2,15 +2,24 @@
 
 Ruby development plug-in for Neovim. Highly unstable.
 
+# Topics
+
+:large_blue_diamond: [Motivation](#motivation)
+:large_blue_diamond: [Installation](#installation)
+:large_blue_diamond: [Setup](#setup)
+:large_blue_diamond: [Commands](#commands)
+
+---
+
```

Comment:
Total detail… but I don't think these horizontal dividers help a lot (same for similar lines). The following heading top margin and bottom border create a nice visual organization already IMHO.

Reply:
I'll fix it